pkg/controller/backup_controller.go

```diff
@@ -436,6 +436,15 @@ func (c *backupController) prepareBackupRequest(backup *velerov1api.Backup) *pkg
 	request.Annotations[velerov1api.SourceClusterK8sMajorVersionAnnotation] = c.discoveryHelper.ServerVersion().Major
 	request.Annotations[velerov1api.SourceClusterK8sMinorVersionAnnotation] = c.discoveryHelper.ServerVersion().Minor
 
+	// Add namespaces with label velero.io/exclude-from-backup=true into request.Spec.ExcludedNamespaces
+	// Essentially, adding the label velero.io/exclude-from-backup=true to a namespace would be equivalent to setting spec.ExcludedNamespaces
+	namespaces, excludeLabel := corev1api.NamespaceList{}, "velero.io/exclude-from-backup"
```

Comment:
This is trivial but it seems we don't need `excludeLabel` as a variable.

Reply:
Would it be better if I move excludeLabel to pkg/controller/constants.go as a const or simply define it as const on the top of the code?